docs/security/how-to-authenticate.md

```diff
@@ -111,6 +115,13 @@ GravitinoClient client = GravitinoClient.builder(uri)
 | `gravitino.authenticator.oauth.serverUri`         | The URI of the default OAuth server.                                                                                                                                                                                                                       | (none)            | Yes if use `oauth` as the authenticator    | 0.3.0            |
 | `gravitino.authenticator.oauth.tokenPath`         | The path for token of the default OAuth server.                                                                                                                                                                                                            | (none)            | Yes if use `oauth` as the authenticator    | 0.3.0            |
+| `gravitino.authenticator.oauth.provider`           | OAuth provider type (default, oidc). Determines the Web UI authentication flow. Use 'oidc' for Web UI OIDC login, 'default' for legacy login or API-only authentication.                                   | `default`         | No                                         | 1.0.0            |
+| `gravitino.authenticator.oauth.clientId`           | OAuth client ID for Web UI authentication.                                                                                                                                                                    | (none)            | No                                         | 1.0.0            |
+| `gravitino.authenticator.oauth.authority`          | OAuth authority/issuer URL for OIDC providers for web UI authentication. (e.g., Azure AD tenant URL).                                                                                                                                   | (none)            | No                                         | 1.0.0            |
+| `gravitino.authenticator.oauth.scope`              | OAuth scopes for Web UI authentication (space-separated).                                                                                                                                                      | (none)            | No                                         | 1.0.0            |
+| `gravitino.authenticator.oauth.jwksUri`            | JWKS URI for server-side OAuth token validation. Required when using JWKS-based validation.                                                                                                                                                               | (none)            | No                                         | 1.0.0            |
+| `gravitino.authenticator.oauth.principalField`     | JWT claim field to use as principal identity (e.g., 'sub', 'client_id', 'appid').                                                                                                                                                                         | `sub`             | No                                         | 1.0.0            |
```

Comment:
`princialFields`? According to https://github.com/apache/gravitino/pull/8049. Do you need add comment here?

Reply:
Updated